docs/UserGuide.md

```diff
@@ -133,8 +149,14 @@ Examples:
 ### View student statistics: `view`
 This would display the following in a pop-up window 
 - the total number of students 
-- the total amount owed by students
-- the number of upcoming exams in the following month
+- the total amount owed by students (Currently shows the exact amount)
+- the number of upcoming exams in following 1 month period (from today up to the same day of the next month)
+
+<box type="tip" seamless>
+Tip:<br>
+Use ths feature to track the total amount of students you're teaching, how much is owed by all of them
+and how plan your schedule based on the number of upcoming exams that will come in the next month
```

Comment:
`Use this feature to track the total amount of students you're teaching, how much is owed by all of them
and plan your schedule based on the number of upcoming exams in the next month!` <- extra exclamation mark for personality wooo!